airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -17,8 +17,188 @@
+- They may be mentioned in a comment in the `.github/CODEOWNERS` file.
+- Suitable candidates for translation ownership may be suggested from engaged translators, upon their consent and approval by the procedure in section 6.1.
+
+## 6. Procedures
+
+### 6.1. Approval process of a translation owner
+
+- The designated code owner should post a thread in the dev. list to request for approval of the translation owner(s) for a supported translation.:
+    - Approvals of a translation owner who is also the code owner can be done by a lazy consensus.
+    - Approvals of translation owners who are non-committers require at least one binding vote of at least 1 PMC member, and no objections from committers/PMC.
+- Within the thread, the code owner should demonstrate that the translation owner is suitable for the role, according to the requirements in section 4.2.
+
+### 6.2. Approval process of a new translation
+
+The following steps outline the process for approving a new translation:
+> [!WARNING]
+> Before merging the PR, the committer should validate that files are in sync with the default language (English).
+
+- Creating a PR to add a new translation to the codebase ([see example](https://github.com/apache/airflow/pull/51258/files)), according to the standard and guidelines, which includes:
+    - The translation files in the `airflow-core/src/airflow/ui/src/i18n/locales/<LOCALE_CODE>` directory, where `<LOCALE_CODE>` is the code of the language (e.g., `fr` for French).
+    - Making the required modifications in `airflow-core/src/airflow/ui/src/i18n/config.ts` ([see example](https://github.com/apache/airflow/pull/51258/files#diff-bfb4d5fafd26d206fb4a545a41ba303f33d15a479d21e0a726fd743bdf9717ff)) (should be done by the code owner).
+    - Updating the `.github/CODEOWNERS` file to include the code owner(s).
+      > [!NOTE]
+      > When assigning a translation sponsor, there should be a comment in the `.github/CODEOWNERS` indicating who the translation owner(s) and the translation sponsor are.
+- Apply procedure 6.1. to approve the translation owner, while announcing the introduction of the new language (including a link to the PR).
+- Only after the steps above are completed, the PR for the new translation could be merged (by the requirements in section 4.7).
+
+### 6.3. Translation conflict resolution
+
+When a conflict arises in a translation-related PR, the following steps will be taken in order:
+
+- The involved parties should first try to reach a consensus through discussion in the PR.
+- If no consensus is reached, a translation owner may decide the outcome.
+- If multiple translation owners are involved and cannot reach consensus, the code owner will decide. If the code owner is sponsored,
+they should base their decision on a neutral source (e.g., a third-party opinion, translation tool, or LLM such as ChatGPT or Claude).
+- If the conflict is between code owners, a PMC member will be involved to resolve the conflict.
```

Comment:
Does this not rather need to follow the VOTING process like in debates about other technical / code questions?

Reply:
I hope there are no real "conflicts". Maybe just "opinons". But languages are usually not so strickt and multiple options are typically a matter of taste.
Anyway I assume PMC role is not needed but it should be rather a binding vote from all committers. Yeah and also if not proficient in the language. Will be the same but less people as PMC as well. Otherwise you could only roll a dice or create a fork of Airflow :-D

Reply:
I am not sure if we should start mailing lists votes on language specific terminology. I don't think this is what the mailing list is for. A simple jusgment call from any PMC member should be sufficient on the PR itself (like any code change we have).

I don't think any code flavor "conflict" between 2 developer over a specific provider would have reach to a mailing list discussion. So in ny prespective it's kinda the same

Reply:
Restored the original phrasing, I'll notify about that in the vote